test/testffmpeg_vulkan.c

```diff
@@ -206,7 +206,7 @@ static int createInstance(VulkanVideoContext *context)
     VkApplicationInfo appInfo = { 0 };
     VkInstanceCreateInfo instanceCreateInfo = { 0 };
     VkResult result;
-    char const *const *instanceExtensions = SDL_Vulkan_GetInstanceExtensions(&instanceCreateInfo.enabledExtensionCount);
+    char const *const *instanceExtensions = SDL_Vulkan_GetInstanceExtensions((int *)&instanceCreateInfo.enabledExtensionCount);
```

Comment:
Oh, right, this is why we used Uint32. We should revert this portion of the commit. Sorry about the churn!

Reply:
I went ahead and took care of this.